install-project.sh

```diff
@@ -165,17 +165,19 @@ elif [[ "$slug" == "apache/servicecomb-pack" ]]; then
 else
     mvn clean install -am -pl $module -DskipTests ${MVNOPTIONS} |& tee mvn-install.log
 fi
-
+echo "mvn clean install -am -pl $module -DskipTests ${MVNOPTIONS} |& tee mvn-install.log" |& tee mvn-install-command.sh
```

Comment:
Can you change the mvn clean install... stuff so that before all the if statements, we set ```command='echo "mvn clean install -am -pl... "'``` and then for specific if conditions that have a slightly different mvn clean install..., such as ```"$slug" == "apache/incubator-dubbo"```, you can set command again so that it correctly does not have the -pl argument.

Then for this echo you can simply ```echo "$command" |& tee mvn-install-command.sh```

Reply:
You are also missing a command to run $command
https://unix.stackexchange.com/questions/356534/how-to-run-string-with-values-as-a-command-in-bash